allenact/algorithms/onpolicy_sync/runner.py

```diff
@@ -76,8 +84,7 @@ def __init__(
         if self.deterministic_cudnn:
             set_deterministic_cudnn()
 
-        if self.seed is not None:
-            set_seed(self.seed)
+        set_seed(self.seed)
```

Comment:
isn't it enough to pass a common seed to the train workers? (when we do that, all models are initialized equally, this was checked with the TODO code block in engine/update(...))